www/src/components/layer-model/layer-model.js

```diff
@@ -94,12 +78,7 @@ const LayerModel = ({
         marginBottom: 6,
```

Comment:
```suggestion
        mb: 6,
```
Can we make it is mb: 6?